src/Http/Http.Extensions/gen/StaticRouteHandlerModel/EndpointParameter.cs

```diff
@@ -57,6 +60,42 @@ public EndpointParameter(IParameterSymbol parameter, WellKnownTypes wellKnownTyp
         }
     }
 
+    private static bool TryGetParsability(IParameterSymbol parameter, WellKnownTypes wellKnownTypes, [NotNullWhen(true)]out Func<string, string, string>? parsingBlockEmitter)
+    {
+        if (parameter.Type.SpecialType == SpecialType.System_String)
+        {
+            parsingBlockEmitter = null;
+            return false;
+        }
+
+        if (ParsabilityHelper.GetParsability(parameter.Type, wellKnownTypes) == Parsability.NotParsable)
+        {
+            parsingBlockEmitter = null;
+            return false;
+        }
+
+        // This is the default TryParse call that we support, but subsequent statements override as appropriate.
+        var preferredTryParseInvocation = (string inputArgument, string outputArgument) => $$"""{{parameter.Type.ToDisplayString(SymbolDisplayFormat.FullyQualifiedFormat)}}.TryParse({{inputArgument}}, out var {{outputArgument}})""";
```

Comment:
If there's an overload that takes an `IFormatProvider`, we should prefer it, so we don't get `CurrentCulture` behavior.